frontend/src/v5/ui/components/shared/modals/templates/alertModal/alertModal.component.tsx

```diff
@@ -20,23 +20,31 @@ import { Button, DialogContent, DialogContentText, DialogTitle } from '@material
 import WarningIcon from '@assets/icons/warning.svg';
 import { FormattedMessage } from 'react-intl';
 import { Container, Actions, Details, Status } from '@/v5/ui/components/shared/modals/modals.styles';
+import { AxiosError } from 'axios';
 
 interface IAlertModal {
 	onClickClose?: () => void,
 	currentActions?: string
 	errorMessage?: string;
-	error?: {
-		request: {
-			response: string;
-		};
-	};
+	error: AxiosError;
 	details?: string
 }
 
 export const AlertModal: React.FC<IAlertModal> = ({ onClickClose, currentActions = '', error, details, errorMessage }) => {
-	const responseData = error?.request?.response ? JSON.parse(error?.request?.response) : {};
-	const { message, status, code } = responseData;
-	const errorStatus = `${status} - ${code}`;
+	let message; let code;
```

Comment:
Do we have any rules that force us to declare variables on different lines?

Reply:
Do you mean ES Lint rules? If so then no